.gitmodules

```diff
@@ -15,4 +15,4 @@
 [submodule "submodules/xdp-for-windows"]
 	path = submodules/xdp-for-windows
 	url = https://github.com/microsoft/xdp-for-windows.git
-	branch = release/1.0
+	branch = dev/daiki/xdp_api_v3
```

Comment:
needs to be a commit in `main`

Reply:
I think main branch cannot be built with msquic
https://github.com/microsoft/xdp-for-windows/issues/741

Reply:
OK, please merge the XDP fix into XDP's main before rebasing this PR onto a main commit.